FindYourRecipe/FindYourRecipe.Web/Pages/Index.razor

```diff
@@ -1,10 +1,123 @@
 ﻿@page "/"
+@inject IJSRuntime JS
+@inject IIngredientService IngredientService
+@inject IRecipeService RecipeService
 
-<PageTitle>Index</PageTitle>
+<PageTitle>Home</PageTitle>
 
-<h1>Hello, world!</h1>
+<div class="container p-5 " style=" text-align:center ;background-color: rgb(225, 215, 198); margin-top: 15px;">
+    <h1 style="font-size: 50px;">Find Your Recipe</h1>
 
-Welcome to your new app.
+    <style>
```

Comment:
I would prefer having the CSS code in a separate file. It works this way as well but it's not necessarily a best practice to do it this way.